Client/Frontend/Browser/URIFixup.swift

```diff
@@ -32,4 +37,10 @@ class URIFixup {
 
         return nil
     }
+
+    static func isURL(string: String?) -> Bool {
+        let regEx = "[-a-zA-Z0-9@:%._\\+~#=]{1,256}\\.[-a-zA-Z0-9@:%._\\+~#=]{2,6}\\b([-a-zA-Z0-9@:%_\\+.~#?&//=]*)"
```

Comment:
I'm wary on using a regular expression like this to check if something is a URL. For regular applications I can see this working fine but since we're a web browser our URL handling should be more strict. Looking at https://mathiasbynens.be/demo/url-regex makes me worry that this might be enough. Usually `NSURL` will try to convert a string to a URL and return something invalid if it can't so maybe try using that to see if what we have is good?